app/models/gobierto_people/person.rb

```diff
@@ -58,5 +60,28 @@ def attributes_for_slug
       [name]
     end
 
+    def events_collection
+      @events_collection ||= GobiertoCommon::Collection.find_by container: self, item_type: 'GobiertoCalendars::Event'
+    end
+
+    def events
```

Comment:
Random comment: ¿Is it a good idea to memoize this? I haven't checked this, but I'm thinking of a situation when after doing `person.reload` the `@events` variable still holds the previous value because it's not an attribute [see the docs](https://apidock.com/rails/ActiveRecord/Base/reload) about the reload method.